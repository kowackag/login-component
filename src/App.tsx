import { Routes, Route } from "react-router-dom";
import Authentication from "./views/Auth";

function App() {
	return (
		<Routes>
			<Route path="/auth" element={<Authentication />} />
		</Routes>
	);
}

export default App;
