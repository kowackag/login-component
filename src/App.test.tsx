import { expect, it, describe } from "vitest";
import { render } from "@testing-library/react";
import App from "./App";
import { BrowserRouter } from "react-router-dom";

describe("SocialIconBox", () => {
	it("App component exists", async () => {
		const wrapper = render(
			<BrowserRouter>
				<App />
			</BrowserRouter>,
		);
		expect(wrapper).toBeTruthy();
		expect(wrapper.container).toBeInTheDocument();
	});

	it("renders correctly", () => {
		const { container } = render(
			<BrowserRouter>
				<App />
			</BrowserRouter>,
		);
		expect(container).toMatchSnapshot();
	});
});
