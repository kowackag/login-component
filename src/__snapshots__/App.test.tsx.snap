// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SocialIconBox > renders correctly 1`] = `<div />`;
